src/components/presentationals/Image/Image.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Image({ src }) {
```

Comment:
What if we want to apply styling to this image or add other image properties? I think it will be better to accept other props as well.
```javascript
function Image({src, ...props}) {
   return <img src={src} {...props} />;
}

Reply:
Done. Thanks